core/src/main/scala/com/ubirch/transformer/actor/KafkaProducerActor.scala

```diff
@@ -0,0 +1,35 @@
+package com.ubirch.transformer.actor
+
+import akka.actor.{Actor, ActorLogging, Props}
+import akka.pattern.pipe
+import com.ubirch.avatar.core.kafka.KafkaProducer
+import com.ubirch.transformer.actor.KafkaProducerActor.KafkaMessage
+
+import scala.concurrent.ExecutionContext
+
+class KafkaProducerActor(kafkaUrl: String, topic: String) extends Actor with ActorLogging {
```

Comment:
I implemented Kafka part as an Actor according to the consistency of this service, which is that each actor has a responsibility of one operation such as MessagePersistenceActor, TransformerOutProducerActor.